presto-cache/src/main/java/com/facebook/presto/cache/alluxio/PrestoCacheContext.java

```diff
@@ -13,21 +13,33 @@
  */
 package com.facebook.presto.cache.alluxio;
 
-import alluxio.client.file.URIStatus;
-import alluxio.wire.FileInfo;
+import alluxio.client.file.CacheContext;
+import alluxio.client.quota.CacheQuota;
+import alluxio.client.quota.CacheScope;
 import com.facebook.presto.hive.HiveFileContext;
+import com.google.common.collect.ImmutableMap;
 
 import static java.util.Objects.requireNonNull;
 
-public class AlluxioURIStatus
-        extends URIStatus
+public class PrestoCacheContext
```

Comment:
HiveFileContext has a incrementCounter() function. The tracked counters are automatically aggregated at the query level. You can simply override CacheContext::incrementCounter() to call HiveFileContext::incrementCounter() to expose the counters to presto.

Reply:
Great, just overrided CacheContext::incrementCounter() to call HiveFileContext::incrementCounter() as you said.  Thank you!